### ISSUE TITLE
Indicate no connection via browser tab icon

### ISSUE THREAD

**Issue by Isgar** [glowing-bear/glowing-bear#539]
If glowing-bear is not connected to weechat (connection lost for some reason) it should somehow indicate it through the browser icon just like the read counter shows the number of messages. Currently i have to navigate to the tab to see that i don't get any more messages.

I would propose an icon with dark eyes if not connected, and glowing with connection.

**Comment by lorenzhs** [comment]
Good idea, we could use favico (if enabled) for that, shouldn't be too hard I hope.

Are you interested in modifying the icon? There's an svg in `assets/img`.

**Comment by Isgar** [comment]
Some ideas with original for comparison (can i display them smaller?):

![glowingbear](https://cloud.githubusercontent.com/assets/1774874/5819713/2c32da66-a0c0-11e4-8437-45c934a11392.png)
![glowingbear-closedeyes-black](https://cloud.githubusercontent.com/assets/1774874/5819715/2c368274-a0c0-11e4-89df-59d90f01c92e.png)
![glowingbear-closedeyes-brown](https://cloud.githubusercontent.com/assets/1774874/5819712/2c2fc86c-a0c0-11e4-9345-8faf6c425fd4.png)
![glowingbear-darkeyes](https://cloud.githubusercontent.com/assets/1774874/5819716/2c38ac7a-a0c0-11e4-9fdc-7bac54efd79a.png)
![glowingbear-greyeyes](https://cloud.githubusercontent.com/assets/1774874/5819714/2c35594e-a0c0-11e4-9fb0-7e1e44037efd.png)
![glowingbear-noglow](https://cloud.githubusercontent.com/assets/1774874/5819717/2c398ec4-a0c0-11e4-912a-33c98018c844.png)
![glowingbear-peaceful](https://cloud.githubusercontent.com/assets/1774874/5819718/2c48d5dc-a0c0-11e4-85f9-39bcbc35bbba.png)
![glowingbear-yellowoff](https://cloud.githubusercontent.com/assets/1774874/5819719/2c4d9db0-a0c0-11e4-8911-f09912eb6b34.png)

**Comment by lorenzhs** [comment]
I like the second-to-last one best, the second one is also quite nice. Good work! Will have to try out how they look at the size of a favicon. I won't get to it before next week, unfortunately.

**Comment by Maffblaster** [comment]
I like the second to last one best as well! Cool icons, @Isgar!

**Comment by lorenzhs** [comment]
Right, I completely forgot to add this, sorry. Hopefully I'll remember this time...
